modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-preparing-playbook-machine-{context}"]
+= Preparing the machine to run the playbook
+
+Before you can add compute machines that use Red Hat Enterprise Linux as the
+operating system to a {product-title} {product-version} cluster, you must
+prepare a machine to run the playbook from. This machine is not part of the
+cluster but must be able to access it.
+
+.Procedure
+
+. Ensure that the `kubeconfig` file for the cluster and the installation program
+that you used to install the cluster are on the machine. One way to accomplish
+this is to use the same machine that you used to install the cluster.
+
+. Configure the machine to access all of the RHEL hosts that you plan to use as
+compute machines. You can use any method that your company allows, including a
+bastion with an SSH proxy or a VPN.
+
+. Configure a user on the machine that you run the playbook on that has SSH
+access to all of the RHEL hosts.
++
+[IMPORTANT]
+====
+If you use SSH key-based authentication, you must manage the key with an
+SSH agent.
+====
+
+. Install the OpenShift Command-line Interface (CLI), commonly known as `oc`, on
+the machine.
+
+. Install Ansible 2.7 and the `OpenShift-Ansible` package on the machine:
+.. If you have not, register the machine with RHSM:
++
+----
+# subscription-manager register --username=<user_name> --password=<password>
+----
+
+.. Pull the latest subscription data from RHSM:
++
+----
+# subscription-manager refresh
+----
+
+.. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*OpenShift*'
+----
+
+.. In the output for the previous command, find the pool ID for an {product-title} subscription and attach it:
++
+----
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Enable the repositories required by {product-title} {product-version}:
++
+----
+# subscription-manager repos \
+    --enable="rhel-7-server-ansible-2.7-rpms" \
+    --enable="rhel-7-server-ose-4.1-rpms"
+----
```

Comment:
some dependency packages for openshift-ansible are needed from rhel7 and rhel7-extra repo